src/collections/_documentation/performance-monitoring/performance/event-detail.md

```diff
@@ -22,18 +22,24 @@ Currently, only root transactions are searchable. Any span data that inherits fr
 
 [{% asset performance/span-details.png alt="Span detail view shows the span id, trace id, parent span id, and other data such as tags." %}]({% asset performance/span-details.png @path %})
 
+You can view span data using either of these methods:
```

Comment:
Let's rephrase this - it's as if you're describing how to get to a different view when you are just explaining how you get to _this_ view. For example: 

`To find these views, you can either go through the [Performance Summary] or [Query Builder]. Event IDs will be linked to open the corresponding Event Detail. `